docs/_toc.yml

```diff
@@ -10,13 +10,13 @@
       - file: publish/netlify
   - file: customize/config
   - file: customize/toc
-  - file: content-types/index
+  - file: file-types/index
```

Comment:
So shall we try adding executablebooks/meta#117 then?
As a "new developer" to JB I just found content-types to be very confusing, since you also have the `content` folder which more accurately is a section on content types

Reply:
yeah I agree the renaming makes more sense 👍 

maybe we could try out the extension and see if it is straightforward to get working?

Reply:
Works, but two annoyances: https://github.com/wpilibsuite/sphinxext-rediraffe/pull/17, https://github.com/wpilibsuite/sphinxext-rediraffe/issues/18

Reply:
These will be merged tomorrow and a new version released 😄